draft-ietf-quic-transport.md

```diff
@@ -1202,28 +1202,27 @@ An endpoint MAY use the following transport parameters:
 initial_max_streams_bidi (0x0002):
 
 : The initial maximum bidirectional streams parameter contains the initial
-  maximum number of application-owned bidirectional streams the peer may
-  initiate, encoded as an unsigned 16-bit integer.  If this parameter is absent
-  or zero, application-owned bidirectional streams cannot be created until a
-  MAX_STREAM_ID frame is sent.  Note that a value of 0 does not prevent the
-  cryptographic handshake stream (that is, stream 0) from being used. Setting
-  this parameter is equivalent to sending a MAX_STREAM_ID
-  ({{frame-max-stream-id}}) immediately after completing the handshake
-  containing the corresponding Stream ID. For example, a value of 0x05 would be
-  equivalent to receiving a MAX_STREAM_ID containing 20 when received by a
-  client or 17 when received by a server.
+  maximum number of bidirectional streams the peer may initiate, encoded as
+  an unsigned 16-bit integer.  Note that the cryptographic handshake stream
+  (that is, stream 0) is exempt, a value of zero does not prevent it from being
```

Comment:
The issue is that stream 0 is still (pending design team) a bidirectional stream initiated by the peer.  So a valid interpretation of this new text would be:

- 0:  Special case, can still use crypto stream
- 1:  One bidirectional stream permitted, which is the crypto stream
- 2:  Two bidirectional streams permitted, one of which is the crypto stream
...and so on.

The point of "application-owned" is not just that zero can't be blocked by this setting, but to clarify that the number available to the application isn't one less than this value due to 0.